src/PrometheusPushGateway/PushGateway.php

```diff
@@ -26,9 +29,9 @@ class PushGateway
     /**
      * PushGateway constructor.
      * @param string $address (http|https)://host:port of the push gateway
-     * @param ClientInterface $client
+     * @param ClientInterface|null $client
      */
-    public function __construct($address, ClientInterface $client = null)
+    public function __construct(string $address, ClientInterface $client = null)
```

Comment:
was thinking, should this be `?ClientInterface` ?

Reply:
yep you are right, will change it.

Reply:
@rdohms i changed it.